virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -77,6 +77,9 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-16-networking"]
 === Networking
 
+//CNV-37037 headless services support
+* You can now xref:../../virt/vm_networking/virt-accessing-vm-internal-fqdn.adoc#virt-accessing-vm-internal-fqdn[access a VM] that is connected to the default internal pod network, on a stable fully qualified domain name (FQDN), by using headless services.
```

Comment:
```suggestion
* You can now use headless services to xref:../../virt/vm_networking/virt-accessing-vm-internal-fqdn.adoc#virt-accessing-vm-internal-fqdn[access a VM] that is connected to the default internal pod network on a stable, fully-qualified domain name (FQDN).
```
Does this change the meaning of the sentence too much? I think it might be easier to read without the clause in commas in the middle.

Reply:
I have removed the commas from the original sentence. Hopefully, that makes it more clear.